be/test/exprs/array_functions_test.cpp

```diff
@@ -5467,4 +5467,55 @@ TEST_F(ArrayFunctionsTest, array_match_only_null) {
         ASSERT_TRUE(dest_column->get(0).get_int8());
     }
 }
+// NOLINTNEXTLINE
+TEST_F(ArrayFunctionsTest, array_contains_seq) {
```

Comment:
here is the case nullable array1 and nullable array2, pls refer to array_contains_no_null to add some cases from nullable array1, not-nullable array2, not-nullable array1, nullable array2, not_nullable array1, not_nullable array2.